livebooks/rtmp/rtmp_sender.livemd

```diff
@@ -40,22 +40,20 @@ defmodule RTMP.Sender.Pipeline do
         location: @video_url,
         hackney_opts: [follow_redirect: true]
       })
-      |> child(:video_parser, %Membrane.H264.FFmpeg.Parser{
-        framerate: {25, 1},
-        alignment: :au,
-        attach_nalus?: true,
-        skip_until_keyframe?: true
+      |> child(:video_parser, %Membrane.H264.Parser{
+        output_alignment: :au,
+        skip_until_keyframe: true,
+        generate_best_effort_timestamps: %{framerate: {25, 1}}
       })
       |> child(:video_realtimer, Membrane.Realtimer)
-      |> child(:video_payloader, Membrane.MP4.Payloader.H264)
+      |> child(:video_payloader, %Membrane.H264.Parser{output_stream_structure: :avc1})
 
     audio_source =
       child(:audio_source, %Membrane.Hackney.Source{
         location: @audio_url,
         hackney_opts: [follow_redirect: true]
       })
       |> child(:audio_parser, %Membrane.AAC.Parser{
-        in_encapsulation: :ADTS,
```

Comment:
I am not a specialist, when it comes to AAC, but make sure, that it is safe, to remove this option

Reply:
In current version of acc parser this field no longer exists. Documentation says, correct acc stream format is supplied with the stream. And it works just fine